__tests__/app.test.js

```diff
@@ -220,3 +221,39 @@ describe("GET/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("returns status:201 and POST comment retruns all the properties of the posted object", () => {
+    const addComment = {
+      author: "rogersop",
+      body: "testComment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(addComment)
+      .expect(201)
+      .then((res) => {
+        const { comment } = res.body;
+
+        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+        expect(comment).toHaveProperty("article_id", 1);
+        expect(comment).toHaveProperty("author", expect.any(String));
+        expect(comment).toHaveProperty("body", expect.any(String));
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+      });
```

Comment:
We should be able to be more specific here with these values, we know what all of these properties should look like. I'd say the created_at date value is the only one that's probably fine to accept as an expect.any(String) as it will change every time we run the test.